### ISSUE TITLE
"afUpdateDemo" needs either "schema" or "collection" attribute

### ISSUE THREAD

**Issue by abtx** [aldeed/meteor-autoform#746]
I'm passing collection=Collections.Projects doc=editedProject. The former returns Projects.find({_id:this._id}), so I'm pointing to the right doc. Any ideas why I'm getting the error above? Cheers!!

**Comment by aldeed** [comment]
Not without seeing a reproduction. Maybe `Collections.Projects` helper is not set to anything.

**Comment by abtx** [comment]
No, I haven't seen the Collections.Projects explicitly to anything. Is this not about having a collection called Projects? Cheers

**Comment by abtx** [comment]
Should the helper return all Projects collection, ie. Projects.find() ?

**Comment by aldeed** [comment]
How are you defining the helper? In some circumstances, it can lose prototype. Or it could be related to code load order.

**Comment by abtx** [comment]
I was defining the helper in the template helpers (Template.name.helpers). This would use _id to get the item from collection that update was attempted on. Other than that I tried to follow the method that uses Template.registerHelper, as in the demo. Neither did work, I changed the method to one shown in Discover Meteor book (I need to do some custom inserts anyway, but was curious to see the AutoForm update in action), so I'll mark this issue as closed. Thanks a bunch for all awesome packages Aldeed!
